com.vivaldi.Vivaldi.yaml

```diff
@@ -0,0 +1,136 @@
+app-id: com.vivaldi.Vivaldi
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+base: org.chromium.Chromium.BaseApp
+base-version: '22.08'
+command: vivaldi
+separate-locales: false
+finish-args:
```

Comment:
indeed with drop the drop access to Chrome/Firefox settings read access.

> Instead of accessing ~/.local/share/applications and icons directories directly, Vivaldi should be using DynamicLauncher portal instead

Ok will admit I am a newb on this front. I am not quite sure what you are saying or how to do this.

> Could ~/.local/lib:create be changed into --persist=.local/lib instead?

I do not think so because that is not a standard directory and does not usually exist but will try again and see.

Reply:
> Ok will admit I am a newb on this front. I am not quite sure what you are saying or how to do this.

It requires Vivaldi supporting this which probably needs upstream changes.

https://flatpak.github.io/xdg-desktop-portal/docs/#gdbus-org.freedesktop.portal.DynamicLauncher

Reply:
persist does not create directories only maps them afaik, see the log posted below. It needs to be the old one.

Reply:
As I understand this directory is created on first startup when vivaldi fetches ffmpeg from somewhere and puts it to that path. Someone could confirm this when using `--persist=.local/lib` with fresh profile.

If that's the case then using persist before app is published would work for all users.